pytorch_lightning/utilities/argparse.py

```diff
@@ -242,9 +242,6 @@ def add_argparse_args(
         if arg == 'track_grad_norm':
             use_type = float
 
-        if arg_default is inspect._empty:
-            arg_default = None
-
```

Comment:
Why is this not needed anymore?

Reply:
Those lines are never reached. The `allowed_types` filter above already avoids the `inspect._empty` case, so it is not possible to trigger the if.